change.js

```diff
@@ -0,0 +1,44 @@
+var CarLot = ( function (Cars) {
+
+// called onkeyup to replace description //
+  CarLot.replaceDescription =  function () {
```

Comment:
You're passing an object to augment, but then you're not augmenting it. Instead you're attaching to the global CarLot object. This works in your code, but only because you're loading this file after `CarLot` has been defined. Switch the order to load this file first and it blows up.